.travis.yml

```diff
@@ -1,20 +1,17 @@
 language: cpp
+sudo: false
 
 compiler: 
 - clang
 - gcc
 
-before_install:
-  - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
-  - sudo apt-get update -qq
-
-install:
-  - sudo apt-get install -qq g++-4.8
-  - sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-4.8 90
-  - sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-4.8 90
-
-  # lua, needed for style checking and possibly later on for bindings generation
-  - sudo apt-get install -qq lua5.1
+addons:
+  apt:
+    sources:
+    - ubuntu-toolchain-r-test
+    packages:
+    - g++-4.8
+    - lua5.1
```

Comment:
The ones we bundle. This asks for their to be a system lua, which we then don't use.